llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -411,6 +417,21 @@ std::string saveModuleIR(Module &M, int I, StringRef Suff) {
   return OutFilename;
 }
 
+bool isImportedFunction(const Function &F) {
+  // Functions with definitions are not imported
+  if (!F.isDeclaration())
+    return false;
+
+  bool ReturnValue = true;
+  if (auto NameStr = itaniumDemangle(F.getName())) {
```

Comment:
[Don't "almost always" use `auto`](https://www.llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable), return type is not immediately obvious here

Reply:
Updated to not use auto.